test/distributed/org/apache/cassandra/fuzz/sai/MultiNodeSAITest.java

```diff
@@ -18,18 +18,66 @@
 
 package org.apache.cassandra.fuzz.sai;
 
+import org.junit.Before;
 import org.junit.BeforeClass;
 
-public class MultiNodeSAITest extends MultiNodeSAITestBase
+import org.apache.cassandra.distributed.Cluster;
+import org.apache.cassandra.distributed.test.sai.SAIUtil;
+import org.apache.cassandra.harry.SchemaSpec;
+
+import static org.apache.cassandra.distributed.api.Feature.GOSSIP;
+import static org.apache.cassandra.distributed.api.Feature.NETWORK;
+
+public class MultiNodeSAITest extends SingleNodeSAITest
 {
     @BeforeClass
     public static void before() throws Throwable
     {
-        MultiNodeSAITestBase.before(false);
+        cluster = Cluster.build()
+                         .withNodes(2)
+                         // At lower fetch sizes, queries w/ hundreds or thousands of matches can take a very long time.
+                         .withConfig(defaultConfig().andThen(c -> c.set("range_request_timeout", "180s")
+                                                                   .set("read_request_timeout", "180s")
+                                                                   .set("native_transport_timeout", "180s")
+                                                                   .set("slow_query_log_timeout", "180s")
+                                                                   .with(GOSSIP).with(NETWORK)))
+                         .createWithoutStarting();
+        cluster.setUncaughtExceptionsFilter(t -> {
+            logger.error("Caught exception, reporting during shutdown. Ignoring.", t);
+            return true;
+        });
```

Comment:
please revert this. This log is not correct, this is called w/e any exception is unhandled and bubbles up to the thread to handle.  We ignore all errors during shutdown, so only non-shutdown errors cause tests to fail... this lambda is saying "we found a bug... YOLO, lets ignore!"

Reply:
This is pre-existing code: https://github.com/apache/cassandra/blob/cep-15-accord/test/distributed/org/apache/cassandra/fuzz/sai/MultiNodeSAITestBase.java#L66

Reply:
cc @maedhroz 

it might be pre-existing but it masks bugs.  The supported pattern is to exclude bugs we know about and don't wish to cause the test to fail, but to ignore all bugs is way too broad...

If @maedhroz approves keeping it we can for the SAI tests

Reply:
@dcapwell is probably right here, and I have no desire to mass-exclude bugs. I don’t imagine it will take that long to find out if there are any we need to exclude specifically?

Reply:
Removed this code; will keep it removed if the tests pass.